src/main/java/bio/terra/cli/businessobject/User.java

```diff
@@ -54,8 +54,17 @@
   // pet SA email that Terra associates with this user. the CLI queries SAM to populate this field.
   private String petSAEmail;
 
-  // Google credentials for the user
-  private UserCredentials userCredentials;
+  // Google credentials for the user. This can be either 1) end-user google credentials from an
+  // oauth browser flow, or 2) end-user or pet sa google credentials pulled from the application
+  // default credentials.
```

Comment:
no. it's not only available in VM. for example, you can set GOOGLE_APPLICATION_CREDENTIALS to the secrive account file on your disk.
Google documentation on how ADC works:
```
If the environment variable GOOGLE_APPLICATION_CREDENTIALS is set, ADC uses the service account key or configuration file that the variable points to.

If the environment variable GOOGLE_APPLICATION_CREDENTIALS isn't set, ADC uses the service account that is attached to the resource that is running your code.

This service account might be a [default service account](https://cloud.google.com/iam/docs/service-accounts#default) provided by Compute Engine, Google Kubernetes Engine, App Engine, Cloud Run, or Cloud Functions. It might also be a [user-managed service account](https://cloud.google.com/iam/docs/service-accounts#user-managed) that you created.

If ADC can't use any of the above credentials, an error occurs.
```

Reply:
Thanks